staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -181,6 +182,11 @@ func (c *RESTClient) Patch(pt types.PatchType) *Request {
 	return c.Verb("PATCH").SetHeader("Content-Type", string(pt))
 }
 
+// Apply begins a PATCH request with the apply patch type.
+func (c *RESTClient) Apply(fieldManager string) *Request {
+	return c.Patch(types.ApplyPatchType).Param("fieldManager", fieldManager)
+}
+
```

Comment:
The REST client follows HTTP nomenclature with its verbs (e.g. Post, Put, ...). I don't see how this helper for apply fits at this level.

I would totally agree to have Apply on the generated client sets.

Reply:
Could you point us to the generated client sets code?

Reply:
Hm I think this is it?

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/generating-clientset.md
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/client-package-structure.md#high-level-client-sets

We would add the tag to all APIs once we determine what the tag should look like.

Reply:
APPLY is a logical verb. I actually think this change is fine and an improvement.

But we need to figure out how to make this easy for the generated clients to express.